src/sknnr/_base.py

```diff
@@ -149,6 +149,12 @@ def kneighbors(
             return_dataframe_index=return_dataframe_index,
         )
 
+    def __sklearn_tags__(self):
```

Comment:
Stupid question, but why did you set `tags.input_tags.sparse` here instead of in `RawKNNRegressor`.  I notice in your matrix that `sparse = True` for `RawKNNRegressor`.  Is that intentional?

Reply:
My rationale was that the minor differences between `RawKNNRegressor` and `KNeighborsRegressor` shouldn't affect sparse data support, so I was pretty confident inheriting that without any further investigation (beyond passing estimator checks). What do you think? Better to set everything `False` to start with?

Reply:
That's kind of what I figured.  We might want to slightly edit the top post in this PR to be:

> * All estimators now explicitly prohibit sparse data (other than `RawKNNRegressor)

I'm fine with keeping what you have as-is.